accounts.md

```diff
@@ -0,0 +1,32 @@
+### Accounts
+
+Please always make sure to turn on 2-Step Verification. It is mandatory.
+
+#### Discord
+Discord is the main communication channel in our engineering team. please install discord on your machine and ask your team lead to give you the necessary permissions.
```

Comment:
You could add a link to our discord, I believe there is a way to request access.

Reply:
Hmm, I didn't find it. Seems to me that an invitation link needs to be shared with the new person. 

But I don't feel like putting a non expiring  invitation link publicly.